studio/components/interfaces/Settings/Logs/LogTable.tsx

```diff
@@ -161,14 +163,11 @@ const LogTable = ({
     }
   }, [stringData])
 
-  const RowRenderer = (props: RowRendererProps<any>) => {
-    return (
-      <Row
-        {...props}
-        className="font-mono tracking-tight cursor-pointer !bg-scale-200 hover:!bg-scale-300"
-      />
+  const RowRenderer = useMemo(() => {
+    return (props: RowRendererProps<any>) => (
+      <Row {...props} isRowSelected={false} selectedCellIdx={undefined} />
     )
-  }
+  }, [])
```

Comment:
small nit: could this be a `useCallback()` instead?

Reply:
nice catch, done!